config/routes.rb

```diff
@@ -13,8 +13,11 @@
   resources :password_resets, only: %i[new create update]
   put 'admin/deals/publish'
   put 'admin/deals/unpublish'
+  put 'admin/collections/publish'
```

Comment:
this is wrong. url should be 'admin/collections/:id/publish', it should be a member action